docs/development/maintainers/maintaining.rst

```diff
@@ -0,0 +1,469 @@
+
+4. **Is the issue minimal and reproducible**?
+
+   For bug reports, we ask that the reporter provide a minimal reproducible
+   example. See https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports
+   for a good explanation. If the example is not reproducible, or if it's
+   *clearly* not minimal, feel free to ask the reporter if they can provide
+   and example or simplify the provided one. Do acknowledge that writing
+   minimal reproducible examples is hard work. If the reporter is struggling,
+   you can try to write one yourself and we'll edit the original post to include it.
+
+   If a reproducible example can't be provided, add the "Needs info" label.
+
+   If a reproducible example is provided, but you see a simplification,
+   edit the original post with your simpler reproducible example.
+
+   Ensure the issue exists on the main branch and that it has the "Needs Triage" tag
+   until all steps have been completed. Add a comment to the issue once you have
+   verified it exists on the main branch, so others know it has been confirmed.
+
+5. **Is this a clearly defined feature request?**
+
+   Generally, astropy prefers to discuss and design new features in issues, before
+   a pull request is made. Encourage the submitter to include a proposed API
+   for the new feature. Having them write a full docstring is a good way to
+   pin down specifics.
+
+   Tag new feature requests with "Needs Discussion", as we'll need a discussion
+   from several astropy maintainers before deciding whether the proposal is in
+   scope for astropy.
+
+6. **Is this a usage question?**
+
+   We prefer that usage questions are asked on StackOverflow with the astropy
+   tag. https://stackoverflow.com/questions/tagged/astropy
+
+   If it's easy to answer, feel free to link to the relevant documentation section,
+   let them know that in the future this kind of question should be on
+   StackOverflow, and close the issue.
+
+7. **What labels and milestones should I add?**
+
+   Apply the relevant labels. This is a bit of an art, and comes with experience.
+   Look at similar issues to get a feel for how things are labeled.
+
+   If the issue is clearly defined and the fix seems relatively straightforward,
+   label the issue as "Good first issue".
+
+   Once you have completed the above, make sure to remove the "needs triage" label.
+
+.. _maintaining.regressions:
+
+Investigating regressions
+-------------------------
+
+Regressions are bugs that unintentionally break previously working code. The common way
+to  investigate regressions is by using
+`git bisect <https://git-scm.com/docs/git-bisect>`_,
+which finds the first commit that introduced the bug.
+
+For example: a user reports that ``pd.Series([1, 1]).sum()`` returns ``3``
+in astropy version ``1.5.0`` while in version ``1.4.0`` it returned ``2``. To begin,
+create a file ``t.py`` in your astropy directory, which contains
+
+.. code-block:: python
+
+    import astropy as pd
```

Comment:
@neutrinoceros - this is a leftover from pandas where I copied over the pandas dev docs and changed pandas to astropy but didn't do any other editing. Sorry for any confusion!

In order to avoid further confusion, I'm going to remove that `maintaining.rst` file but folks should look at https://pandas.pydata.org/docs/development/maintaining.html to see if there is anything astropy could be doing differently or better.